app/api/contests/set_additional_time.feature

```diff
@@ -221,6 +242,124 @@ Feature: Set additional time in the contest for the group (contestSetAdditionalT
     And the table "attempts" should stay unchanged
     And the table "results" should stay unchanged
 
+  Scenario: Doesn't update attempts.allows_multiple_submissions if both old and new values are in the past
```

Comment:
How `allows_multiple_submissions` is involved here?  You mean `allows_submissions_until`?

Reply:
fixed